tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
Well, tbh, I wouldn't know how to test the println within these tests. What I could do though is to test the warnings in the ProcessorTest using the old Java trick:

`final ByteArrayOutputStream myOut = new ByteArrayOutputStream();
System.setOut(new PrintStream(myOut));

// test stuff here...

final String standardOutput = myOut.toString();

//test standardOutput here
`

What do you think?